### DISCUSSION TITLE
Passing Name and the Language of the user initially

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/sample-app-aoai-chatGPT#666] [Q&A]
I have integrated this chatbot inside my application through iFrame. So basically my app is deployed on an AppService and the chatbot is deployed onto a different AppService. 

What I want to achieve is that, when the user opens the chatbot from the application, is there a way for me to pass the name and the language of that particular user, so the chatbot will initially greet the user with his/her name and continue to chat with that desired language instead of explicitly mentioning it from the user's side. 

Is that possible as I'm loading the chatbot via an Iframe inside my application?

**Comment by mmdmirh** [comment]
Hi,
does your app have access to /.auth/me endpoint? if so, you can parse the required info from the response.
